package.json

```diff
@@ -43,6 +44,7 @@
     "CHANGELOG.md"
   ],
   "scripts": {
+    "debug": "http-server --port 3000 -c-1",
```

Comment:
Feel free to open up a separate PR to introduce this debug server @jessetane. It's going to get more relevant as we merge PRs like #8838.